code/go/0chain.net/smartcontract/dbs/event/enums.go

```diff
@@ -96,6 +96,11 @@ const (
 	TagUpdateAuthorizerTotalStake
 	TagUpdateAuthorizerTotalUnStake
 	TagUniqueAddress
+	TagMinerHealthCheck
+	TagSharderHealthCheck
+	TagBlobberHealthCheck
+	TagAuthorizerHealthCheck
+	TagValidatorHealthCheck
```

Comment:
do we emit it anywhere?

Reply:
Miner => https://github.com/0chain/0chain/blob/a95626e9f01c2a020101c8b6d364bce8bb042d56/code/go/0chain.net/smartcontract/minersc/miner_eventdb.go#L123
Sharder => https://github.com/0chain/0chain/blob/a95626e9f01c2a020101c8b6d364bce8bb042d56/code/go/0chain.net/smartcontract/minersc/sharder_eventdb.go#L120
Blobber => https://github.com/0chain/0chain/blob/a95626e9f01c2a020101c8b6d364bce8bb042d56/code/go/0chain.net/smartcontract/storagesc/blobber_eventdb.go#L108
For Authorizer and Validator, they still don't have front-end for health check mechanism so I left them to be used when this is implemented